### ISSUE TITLE
Application crashes when trying to play a song after a fresh npm install

### ISSUE THREAD

**Issue by anotheredward** [96AA48/yplayer#2]
running `yplayer -s "hurt Johnny Cash"` after a fresh `npm install -g yplayer` results in the following:
```
fs.js:500
  return binding.open(pathModule._makeLong(path), stringToFlags(flags), mode);
                 ^
Error: ENOENT, no such file or directory '/home/ed/.yplayerrc'
    at Error (native)
    at Object.fs.openSync (fs.js:500:18)
    at Object.fs.writeFileSync (fs.js:1099:15)
    at settings (/Users/ed/.nvm/versions/node/v0.12.2/lib/node_modules/yplayer/app.js:53:8)
    at Object.<anonymous> (/Users/ed/.nvm/versions/node/v0.12.2/lib/node_modules/yplayer/app.js:18:3)
    at after (/Users/ed/.nvm/versions/node/v0.12.2/lib/node_modules/yplayer/node_modules/cli/cli.js:1011:18)
    at Object.cli.main (/Users/ed/.nvm/versions/node/v0.12.2/lib/node_modules/yplayer/node_modules/cli/cli.js:1016:9)
    at Object.<anonymous> (/Users/ed/.nvm/versions/node/v0.12.2/lib/node_modules/yplayer/app.js:17:5)
    at Module._compile (module.js:460:26)
    at Object.Module._extensions..js (module.js:478:10)
```

**Comment by 96AA48** [comment]
Seems odd, with the first run it should show something along the lines of `ERROR: Go to ~/.yplayerrc and add your email and password`. You haven't seen anything along those lines?

**Comment by 96AA48** [comment]
@anotheredward I'm working on a new version that doesn't need an apikey or the `~/.yplayerrc` file, so the issue should be fixed by then. Stay tuned!

**Comment by anotheredward** [comment]
:+1: Hit me up when you're done and I'll happily test it :)

**Comment by 96AA48** [comment]
I reworked the module a bit. You don't need a Youtube API key from now on! I also added support to watch videos from the cli as well. Feel free to test it out with the new v0.2.0 release on npm. Be sure to delete your `.yplayerrc` before starting up!

**Comment by anotheredward** [comment]
At version 0.2.2:
fs.js:549
  return binding.open(pathModule._makeLong(path), stringToFlags(flags), mode);
                 ^

Error: ENOENT: no such file or directory, open '/home/ed/.yplayerrc'
    at Error (native)
    at Object.fs.openSync (fs.js:549:18)
    at Object.fs.writeFileSync (fs.js:1161:15)
    at settings (/Users/ed/.nvm/versions/node/v4.2.1/lib/node_modules/yplayer/app.js:53:8)
    at Object.<anonymous> (/Users/ed/.nvm/versions/node/v4.2.1/lib/node_modules/yplayer/app.js:22:3)
    at after (/Users/ed/.nvm/versions/node/v4.2.1/lib/node_modules/yplayer/node_modules/cli/cli.js:1011:18)
    at Object.cli.main (/Users/ed/.nvm/versions/node/v4.2.1/lib/node_modules/yplayer/node_modules/cli/cli.js:1016:9)
    at Object.<anonymous> (/Users/ed/.nvm/versions/node/v4.2.1/lib/node_modules/yplayer/app.js:21:5)
    at Module._compile (module.js:435:26)
    at Object.Module._extensions..js (module.js:442:10)

**Comment by 96AA48** [comment]
Ah, well. This was actually an issue with the sister program of this called [gmplayer](http://npmjs.org/gmplayer). I'll patch yplayer with the same solution and publish it as 0.2.3, stay tuned!

**Comment by 96AA48** [comment]
@anotheredward Issue should be resolved, let me know if it works and I'll close the issue!

**Comment by anotheredward** [comment]
:+1: It works! (on some songs, will triage and file additional bug, guessing it's to do with the audio not being an mp3!?)

**Comment by 96AA48** [comment]
Good to know that the issue has been resolved. Feel free to open another issue for the bug you're running into right now or post it in this thread. Thanks!
